tests/cypress/plugins/index.js

```diff
@@ -0,0 +1,18 @@
+
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+module.exports = (on, config) => {
```

Comment:
Just as a note prior to review, these stub files are not doing anything, but are recreated on every run of Cypress if they are not there (as are the folders). I would suggest to merge them as such to prevent this behavior.